### PR TITLE
Finished nav bar  + added an example of a recipe slip

### PR DESCRIPTION
**Pull request by Migu3lito5**
- In the /home route I created a nav bar that will be utilized accross the pages with a nav bar.

-  created some sample code to display how we will organize and display the data onto the user when viewing a recipe. 

- Also created a trim to the top of the page

Screenshot 2022-11-23 194034

### CODE REVIEW

**Comment by yuukiok** [comment]
[MUST] could you fix the merge conflict? now I cannot merge this pull request.
I think this command would be work. also could you add the command to readme if it works.
```
git checkout main
git pull origin main
git checkout {branch that you want to resolve conflict}
git merge main
```

**Comment by Migu3lito5** [comment]
fixed issue with index.js file. Let me know when i can merge.

File: views/home.ejs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      <div class="body-row active"> <i class="fa-sharp fa-solid fa-house"></i><a href="">Home</a> </div>
+      <div class="body-row"><i class="fa-solid fa-bookmark"></i><a href="">Saved</a></div>
+      <div class="body-row"><i class="fa-solid fa-gear"></i><a href="">Info</a></div>
+    </section>
+
+    <section class="nav-footer">
+      <div class="body-row"><i class="fa-solid fa-right-from-bracket"></i><a href="">Logout</a></div>
+    </section>
+
+  </nav>
```
Comment (yuukiok) [inline views/home.ejs]: [IMO] I think you can use this navigation for all pages Home/Saved/Info, so to change this navigation as component is better.

File: views/home.ejs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  <div class="trim"></div>
+  <nav>
+    <section class="nav-header">
+      <h1>Welcome <br> User!</h1>
+    </section>
+
+    <section class="nav-body">
+      <div class="body-row active"> <i class="fa-sharp fa-solid fa-house"></i><a href="">Home</a> </div>
+      <div class="body-row"><i class="fa-solid fa-bookmark"></i><a href="">Saved</a></div>
+      <div class="body-row"><i class="fa-solid fa-gear"></i><a href="">Info</a></div>
```
Comment (yuukiok) [inline views/home.ejs]: [SHOULD] class of nav-body can be a tag to change the link.

**Pull request merged**